### PR TITLE
fixes to the calendar

### PR DESCRIPTION
**Pull request by fullyouu**

### CODE REVIEW

**Comment by fullyouu** [comment]
Screen Shot 2021-12-02 at 6 23 38 PM
After long discussions, it seems like common color for all slots is best

File: app/assets/stylesheets/components/_calendar.scss [hunk 1]
```diff
@@ -126,26 +126,32 @@
...
+      font-size: 11px;
 
       &--next,
       &--previous {
         background-color: hsla(238, 73%, 70%, 0.4);
       }
     }
     &-late {
-      background-color: hsla(238, 73%, 50%, 0.9);
+      background-color: hsla(238, 73%, 70%, 0.9);
```
Comment (ThomasGerleve) [inline app/assets/stylesheets/components/_calendar.scss:138]: i would want to keep it color-coded ¯\_(ツ)_/¯

File: app/assets/stylesheets/components/_calendar.scss [hunk 2]
```diff
@@ -126,26 +126,32 @@
...
 
       &--next,
       &--previous {
-        background-color: hsla(238, 73%, 70%, 0.4);
+        background-color: $purple;
       }
     }
     &-night {
-      background-color: hsla(238, 73%, 30%, 0.9);
+      background-color: hsla(238, 73%, 70%, 0.9);
```
Comment (ThomasGerleve) [inline app/assets/stylesheets/components/_calendar.scss:150]: same

File: app/assets/stylesheets/components/_calendar.scss [hunk 3]
```diff
@@ -163,10 +169,14 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    background-color: rgb(218, 126, 64);
+    background-color: $organe;
```
Comment (ThomasGerleve) [inline app/assets/stylesheets/components/_calendar.scss:172]: organe or orange?

File: app/assets/stylesheets/config/_colors.scss [hunk 4]
```diff
@@ -13,4 +13,6 @@ $light-purple: #D3D5F8;
 $pink: #FF84FF;
 $dark-petrol: #0c9cbe;
 $petrol: #00BFBA;
-$light-petrol: #9fc5c4;
+$light-petrol: #bcf8f6;
+$emergency-rose: #F18F8E;
+$organe: rgb(224, 174, 80);
```
Comment (ThomasGerleve) [inline app/assets/stylesheets/config/_colors.scss:18]: i would go for orange :)

File: app/views/user_shifts/_calendar.html.erb [hunk 5]
```diff
@@ -114,7 +116,7 @@
           data: {
             action: "click->calendar#navigate"
           } do %>
-          <i class="fas fa-caret-square-left"></i>
+          <i class="fas fa-caret-square-left arrow"></i>
```
Comment (ThomasGerleve) [inline app/views/user_shifts/_calendar.html.erb:119]: would go for calendar-arrow, just to be more specific

File: app/views/user_shifts/_calendar.html.erb [hunk 6]
```diff
@@ -125,7 +127,7 @@
           data: {
             action: "click->calendar#navigate"
           } do %>
-          <i class="fas fa-caret-square-right"></i>
+          <i class="fas fa-caret-square-right arrow"></i>
```
Comment (ThomasGerleve) [inline app/views/user_shifts/_calendar.html.erb:130]: would go for calendar-arrow, just to be more specific

File: app/assets/stylesheets/components/_calendar.scss [hunk 7]
```diff
@@ -163,10 +169,14 @@
...
 
     &--emergency {
     background-color: rgb(218, 64, 64);
     }
   }
 }
+
+.arrow {
```
Comment (ThomasGerleve) [inline app/assets/stylesheets/components/_calendar.scss:180]: would go for calendar-arrow, just to be more specific

**Pull request closed without merging**